install_config/router/default_haproxy_router.adoc

```diff
@@ -952,6 +952,67 @@ level deep. So in this example, the certificate would be valid for
 _a.cloudapps.example.com_ but not for _a.b.cloudapps.example.com_.
 ====
 
+[[manually-redeploy-certs]]
+== Manually Redeploy Certificates
+
+To manually redeploy the router certificates:
+
+. Check to see if a secret containing the default router certificate was added to the router: 
++
+----
+$ oc volumes dc/router 
+
+deploymentconfigs/router
+  secret/router-certs as server-certificate
+    mounted at /etc/pki/tls/private
+----
++
+If the certificate is added, skip the following step and overwrite the secret. 
+
+. Make sure that you have a default certificate directory set for the following variable `DEFAULT_CERTIFICATE_DIR`: 
++
+----
+$ oc env dc/router --list
+
+DEFAULT_CERTIFICATE_DIR=/etc/pki/tls/private
+----
++
+If not, create the directory using the following command: 
++
+----
+$ oc env dc/router DEFAULT_CERTIFICATE_DIR=/etc/pki/tls/private
+----
+
+. Export the certificate to PEM format:
++
+----
+$ cat custom-router.crt custom-ca.crt > custom-router.pem 
+----
+
+. Overwrite or create a router certificate secret: 
++
+If the certificate secret was added to the router, overwrite the secret. If not, create a new secret. 
++
+To overwrite the secret, run the following command:
++
+---- 
+$ oc secrets new router-certs tls.crt=custom-router.crt tls.key=router.key -o json --type='kubernetes.io/tls' --confirm | oc replace -f -
```

Comment:
sorry, maybe I missed this.
here `tls.key=router.key` better to use 'tls.key=custom-router.key' in order to keep consistent with 'custom-router.crt' since they are belong to a pair of key.